doc/bitmasks.rst

```diff
@@ -161,6 +196,16 @@ BADREADNOISE                 30 Bad read noise in one of the 3 cameras
 RESERVED31                   31 Reserved sign bit; do not use
 ==================== ========== ===========
 
+**Notes**:
+
+  * Bit 3 (RESTRICTED) is informative and doesn't necessarily mean that the spectrum is bad,
```

Comment:
It isn't necessary to indent lists in RST, and it could interfere with the built-in formatting.